caffe2/utils/knob_patcher.cc

```diff
@@ -1,4 +1,7 @@
-// (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+// (c) Meta Platforms, Inc. and its affiliates.
```

Comment:
I know we're getting nitpicky, but for consistency with the world, our open source headers look slightly different than just removing 'Confidential and proprietary'. You got most of them right, but this and a couple others are wrong.
```suggestion
// // Copyright (c) Meta Platforms, Inc. and affiliates.
```